### PR TITLE
Move Tencent.qq-devtool 0.70.2209190 to Tencent.QQDevTools 0.70.2209190

### PR DESCRIPTION
**Pull request by SpecterShell**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171587)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
InstallerUrl contains Manifest version instead of PR version:
  InstallerUrl: https://miniapp.gtimg.cn/qqdevtools/online/qq_devtools_0.70.2209190_x64.exe

(Automated message - build 894)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Installers:
+- Architecture: x64
+  InstallerUrl: https://miniapp.gtimg.cn/qqdevtools/online/qq_devtools_0.70.2209190_x64.exe
+  InstallerSha256: 234C860E8045E3037F491E0047B7D13A351698105D4235B1F19102267AD2D7CB
+  AppsAndFeaturesEntries:
+    - DisplayName: QQ小程序开发者工具 version 0.70.2209190_x64
+      DisplayVersion: 0.70.2209190_x64
+      ProductCode: '{3BB08405-B515-42FE-B903-325CE9A8208C}_is1'
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (ItzLevvie) [inline manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.installer.yaml:19]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ShortDescription: develop QQ webpages and mini programs
+Tags:
+- development
+- mini-program
+- nodejs
+- nwjs
+- qq
+- web
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (ItzLevvie) [inline manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.locale.en-US.yaml:21]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Tencent.QQDevTools
+PackageVersion: 0.70.2209190
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (ItzLevvie) [inline manifests/t/Tencent/QQDevTools/0.70.2209190/Tencent.QQDevTools.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**